lib/app_server.py

```diff
@@ -17,7 +19,7 @@
 from lib.tarantool_server import Test
 from lib.tarantool_server import TarantoolServer
 from lib.tarantool_server import TarantoolStartError
-from lib.utils import find_port
+from lib.utils import find_port, bytes_to_str
```

Comment:
Please, keep the style: one import object per line.

Reply:
Ok, I'll change it.
FYI, I relied on this document
https://www.tarantool.io/en/doc/latest/dev_guide/python_style_guide/#imports